ompi/errhandler/errhandler.h

```diff
@@ -227,10 +228,20 @@ struct ompi_request_t;
  */
 #define OMPI_ERRHANDLER_INVOKE(mpi_object, err_code, message) \
```

Comment:
Just reviewed this again. We can't just pass NULL because the macro expands to (mpi_object)->field, which is syntactically incorrect code even in the else case of `if(NULL != (mpi_object))`. The whole point of the macro is to do some sort of macro-based type polymorphism on `mpi_object` so it cannot be replaced by a function either.

We could have a macro that does not take the mpi_object argument, as an alternative to mark that we do not know what the mpi_object is.

Reply:
That change has been implemented.